src/memory.rs

```diff
@@ -57,23 +64,33 @@ impl Buffer {
     }
 }
 
+impl Default for Buffer {
```

Comment:
I made this explicit so that it is clearly auditable that the pointer is initialized as null. I depend on this property later when determining whether or not Go wrote to this memory. if you think the `derive` is enough, then sure, i'll change it. I'm pretty sure it sets a null pointer as well.

Reply:
Okay, it does set null. But if it makes you more secure, then leave this one. No problem.